trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Lift/vortex_lift.py

```diff
@@ -68,9 +70,10 @@ def vortex_lift(state,settings,geometry):
             # Apply to wing lift
             wing_lift[Mc < 1.0] = vortex_cl[Mc < 1.0]
         
-        wings_lift += wing_lift
+        # THIS CANNOT WORK
+        #wings_lift += wing_lift
```

Comment:
Does this need to be made an issue and resolved? If so create one

Reply:
@timdmacdo, please provide your thoughts. Can this be made to work arbitrarily?

Reply:
The version I currently use does this. What's the explanation for saying it can't work?

Reply:
@planes just bringing your attention to this